client/src/styles/App.scss

```diff
@@ -144,15 +142,9 @@
   }
 
   h1.page-title {
-    margin: 0;
+    @include mobile-h2;
```

Comment:
also, don't love that the most important h1 in our site hierarchy  is styled as a mobile-h2 but alas

Reply:
ugh yeah, can't wait to just update all the type styles for design system eventually